src/ape/api/providers.py

```diff
@@ -20,12 +20,23 @@
 
 
 class TransactionType(Enum):
+    """
+    Transaction Enumerables type constants defined by EIP-1559.
```

Comment:
Technically this is EIP-2718, not EIP-1559 that defined this

https://eips.ethereum.org/EIPS/eip-2718